package/boot/uboot-rockchip/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2013-2016 OpenWrt.org
```

Comment:
Remove?
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Reply:
OK, I've removed the copyright headers.